addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -251,6 +260,30 @@ odoo.define('website_slides.quiz', function (require) {
             });
         },
 
+        /**
+         * Will check if we have answers coming from the session and re-apply them.
+         */
+        _checkSessionAnswers: function () {
+            if (!this.slide.answers || this.slide.answers.length === 0) {
+                return;
+            }
+
+            var self = this;
+            this.$('.o_wslides_js_lesson_quiz_question').each(function () {
+                var $question = $(this);
```

Comment:
useless variable

Reply:
As we use $(this) multiple times in this block, I think it's cool to give some context.
So I'm keeping it :)

Reply:
you have raison